### PR TITLE
planet colonizer added

### LINKED ISSUE #4287: [New game]: Planet colonizer
### 🎮 Game Request

Game logic and basic description
is a simple and engaging web-based game where players are tasked with colonizing planets by clicking on them. The game is built using HTML, CSS, and JavaScript, providing an interactive experience right in the browser.

### Point down the features

# Game points
- Interactive Gameplay: Players can click on planets to colonize them.
- Visual Feedback: Colonized planets change color to indicate they have been successfully colonized.
- Score Tracking: A scoreboard keeps track of the number of planets colonized by the player.
 # How to Play:
- Open the Game: Launch the game by opening the index.html file in your web browser.
- Colonize Planets: Click on any of the planets displayed on the screen. Each click will change the planet's color, indicating that it has been colonized.
- Track Your Progress: The scoreboard at the bottom of the screen will update to show the number of planets you have colonized.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Yashgabani845**
## PR Description 📜

added planet colonizer 
added required 

1. html,css file
2. game logic in js
3. readme file for game
4.  game entry in readme and json
5.  added image in assets

Fixes #4287 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸
Screenshot (473)

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Yashgabani845 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by Yashgabani845** [comment]
@ishita-43  kindly review this PR
i have one evry steps of guideline and game as well 
thank you for opportunity

File: assets/js/gamesData.json [hunk 1]
```diff
@@ -409,6 +409,10 @@
     "gameTitle": "Rubik's Solver",
     "gameUrl": "Rubik's_solver",
     "thumbnailUrl": "cube_Solver.webp"
+  },"499":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:412]: remove this from here!!

File: assets/js/gamesData.json [hunk 2]
```diff
@@ -2048,7 +2052,7 @@
   "gameTitle": "Virtual Pet Game",
   "gameUrl": "Virtual_Pet_Game",
   "thumbnailUrl": "Virtual_Pet_Game.png"
```
Comment (ishita-43) [inline assets/js/gamesData.json:2054]: and place your game after this entry with no 82

File: assets/js/gamesData.json [hunk 3]
```diff
@@ -2048,7 +2052,7 @@
...
-
+},"511":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2055]: remove this all

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Hoping youwill make desired changes soon!!

**Comment by kunjgit** [comment]
Thank you  @Yashgabani845 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**